### PR TITLE
docs: suggestions from review

### PR DESCRIPTION
**Pull request by sohkai**
~~Will add more commits as I go through the documentation, but please feel free to review already made changes!~~

All suggestions added!

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -2,10 +2,11 @@
 
 Welcome to the Aragon Connect documentation.
 
-* The **Guides** section contains guides that will help you build a dapp using Aragon Connect.
-* The **Advanced** section contains guides that are helpful if you want to write your own app connector, or if you want to have a deeper understanding on how things work.
-* The **Connectors** section contains in-depth documentation of the Connectors availables.
-* The **API Reference** section contains a complete documentation of the functions and objects exposed by Aragon Connect.
+* The [**Guides**](./guides) section contains guides that will help you build a dapp using Aragon Connect.
+* The [**Advanced**](./advanced) section contains guides that are helpful if you want to write your own app connector or want to gain a deeper understanding of everything under the hood.
+* The [**Connectors Reference**](./connectors-reference) section contains in-depth documentation of the [Connectors](./api-reference/connectors.md) available.
```
Comment (bpierre) [inline docs/README.md:7]: I was thinking of using “Connectors” for this section: I would like each of these to also contain a “how to use it” with examples, which some of them have already.

File: docs/guides/getting-started.md [hunk 2]
```diff
@@ -64,16 +65,16 @@ yarn add @aragon/connect-thegraph-voting
 yarn add @aragon/connect-thegraph-tokens
 ```
 
-See “Fetching data from an app state” below to understand how to use these app connectors.
+See [“Fetching an app's state”](#fetching-an-apps-state) below to understand how to use these app connectors.
```
Comment (bpierre) [inline docs/guides/getting-started.md:68]:
```suggestion
See [“Fetching an app’s state”](#fetching-an-apps-state) below to understand how to use these app connectors.
```

File: docs/guides/getting-started.md [hunk 3]
```diff
@@ -83,21 +84,22 @@ It returns an [`Organization`](../api-reference/organization.md) instance.
...
 ```javascript
+// Connect to the Rinkeby test network
 const org = await connect('example.aragonid.eth', 'thegraph', { chainId: 4 })
 ```
 
-Note: other than the Ethereum main network \(default\), only [Rinkeby](https://docs.ethhub.io/using-ethereum/test-networks/#rinkeby) is supported by the `thegraph` connector at the moment.
+Note: other than the Ethereum main network \(default\), only [Rinkeby](https://docs.ethhub.io/using-ethereum/test-networks/#rinkeby) and [xDAI](https://www.xdaichain.com/) are supported by the `thegraph` connector at the moment.
 
-## Fetching data from an app state
+## Fetching an app's state
```
Comment (bpierre) [inline docs/guides/getting-started.md:98]:
```suggestion
## Fetching an app’s state
```

File: docs/guides/getting-started.md [hunk 4]
```diff
@@ -166,31 +168,30 @@ handler.unsubscribe()
...
 for (const transaction of path.transactions) {
   await ethers.sendTransaction(transaction)
 }
 ```
 
 ## Going further
 
-Once you are familiar with the basics of Aragon Connect, you may want to start exploring the [API documentation](../api-reference/connect.md) and the [examples](https://github.com/aragon/connect/tree/master/examples) provided in the repository.
-
+Now that you are familiar with the basics of Aragon Connect, you may want to explore the [examples provided in the repository](https://github.com/aragon/connect/tree/master/examples) and [API documentation](../api-reference/).
```
Comment (bpierre) [inline docs/guides/getting-started.md:197]:
```suggestion
Now that you are familiar with the basics of Aragon Connect, you may want to explore the [examples provided in the repository](https://github.com/aragon/connect/tree/master/examples) and the [API documentation](../api-reference/).
```

File: docs/api-reference/app.md [hunk 5]
```diff
@@ -1,48 +1,47 @@
...
 | `isForwarder` | `Boolean` | Whether the app can act as a forwarder. |
 | `isUpgradeable` | `Boolean` | Whether the app can be upgraded. |
-| `kernelAddress` | `String` | The address of the kernel. |
+| `kernelAddress` | `String` | The address of the kernel this app is installed on. |
 | `name` | `String` | Name of the app, from the repository. E.g. `"Tokens"`. |
-| `registryAddress` | `String` | Address of the aragonPM registry for this app. |
-| `registry` | `String` | Name of the aragonPM registry for this app. E.g. `"aragonpm.eth"` |
-| `repoAddress` | `String` | Address of the Repo the app was installed from. |
-| `sourceUrl` | `String` | URL of the app source code. |
+| `registryAddress` | `String` | Address of the aragonPM registry this app's repo is published to. |
```
Comment (bpierre) [inline docs/api-reference/app.md:25]:
```suggestion
| `registryAddress` | `String` | Address of the aragonPM registry this app’s repo is published to. |
```

File: docs/api-reference/app.md [hunk 6]
```diff
@@ -1,48 +1,47 @@
...
+| `registry` | `String` | Name of the aragonPM registry this app's repo is published to. E.g. `"aragonpm.eth"` |
+| `repoAddress` | `String` | Address of the aragonPM repository the app was installed from. |
+| `sourceUrl` | `String` | URL of the app's source code. |
```
Comment (bpierre) [inline docs/api-reference/app.md:28]:
```suggestion
| `sourceUrl` | `String` | URL of the app’s source code. |
```

File: docs/api-reference/app.md [hunk 7]
```diff
@@ -1,48 +1,47 @@
...
 | `isUpgradeable` | `Boolean` | Whether the app can be upgraded. |
-| `kernelAddress` | `String` | The address of the kernel. |
+| `kernelAddress` | `String` | The address of the kernel this app is installed on. |
 | `name` | `String` | Name of the app, from the repository. E.g. `"Tokens"`. |
-| `registryAddress` | `String` | Address of the aragonPM registry for this app. |
-| `registry` | `String` | Name of the aragonPM registry for this app. E.g. `"aragonpm.eth"` |
-| `repoAddress` | `String` | Address of the Repo the app was installed from. |
-| `sourceUrl` | `String` | URL of the app source code. |
+| `registryAddress` | `String` | Address of the aragonPM registry this app's repo is published to. |
+| `registry` | `String` | Name of the aragonPM registry this app's repo is published to. E.g. `"aragonpm.eth"` |
```
Comment (bpierre) [inline docs/api-reference/app.md:26]:
```suggestion
| `registry` | `String` | Name of the aragonPM registry this app’s repo is published to. E.g. `"aragonpm.eth"` |
```

File: docs/api-reference/app.md [hunk 8]
```diff
@@ -1,48 +1,47 @@
...
+| `repoAddress` | `String` | Address of the aragonPM repository the app was installed from. |
+| `sourceUrl` | `String` | URL of the app's source code. |
 | `version` | `String` | The current version of the app. |
 
 ## Methods
 
 ### App\#repo\(\)
 
-Fetch the app repository.
+Fetch the app's aragonPM repository.
```
Comment (bpierre) [inline docs/api-reference/app.md:35]:
```suggestion
Fetch the app’s aragonPM repository.
```

File: docs/api-reference/app.md [hunk 9]
```diff
@@ -1,48 +1,47 @@
...
 
 | Name | Type | Description |
 | :--- | :--- | :--- |
-| returns | `Promise<Repo>` | A promise resolving to the app repository. |
+| returns | `Promise<Repo>` | A promise resolving to the app's aragonPM repository. |
```
Comment (bpierre) [inline docs/api-reference/app.md:39]:
```suggestion
| returns | `Promise<Repo>` | A promise resolving to the app’s aragonPM repository. |
```

File: docs/api-reference/app.md [hunk 10]
```diff
@@ -1,48 +1,47 @@
...
 
 ### App\#roles\(\)
 
-Fetch the app roles.
+Fetch the app's roles.
```
Comment (bpierre) [inline docs/api-reference/app.md:43]:
```suggestion
Fetch the app’s roles.
```

File: docs/api-reference/app.md [hunk 11]
```diff
@@ -1,48 +1,47 @@
...
 
 | Name | Type | Description |
 | :--- | :--- | :--- |
-| returns | `Promise<Role[]>` | A promise resolving to the app roles. |
-
+| returns | `Promise<Role[]>` | A promise resolving to the app's roles. |
```
Comment (bpierre) [inline docs/api-reference/app.md:47]:
```suggestion
| returns | `Promise<Role[]>` | A promise resolving to the app’s roles. |
```

File: docs/guides/getting-started.md [hunk 12]
```diff
@@ -183,10 +183,10 @@ Then to retrieve the path you want, pass the account that will be signing the tr
 const [path] = intent.paths(wallet.account)
 ```
 
-Finally, you can sign the different transactions associated to this path. Aragon Connect doesn’t handle any signing itself, but returns an object that is ready to use with the library of your choice: [Web3.js](https://web3js.readthedocs.io/en/1.0/), [Ethers](https://docs.ethers.io/v5/), or even a JSON-RPC connection to an Ethereum node.
+Finally, you can sign the different transactions associated to this path. Aragon Connect doesn’t handle any signing itself, but returns an object that is ready to use with the library of your choice: [ethers.js](https://docs.ethers.io/v5/), [Web3.js](https://web3js.readthedocs.io/en/1.0/), or even a [JSON-RPC connection to an Ethereum node](https://eips.ethereum.org/EIPS/eip-1474).
 
 ```javascript
-// Sign the transactions using Ethers
+// Sign the transactions using ethers.js
```
Comment (bpierre) [inline docs/guides/getting-started.md:189]: I think they are now using the name “Ethers” or “ethers” for the library, and use “ethers.js” for the GitHub repo name.

See https://twitter.com/ethersproject and https://github.com/ethers-io/ethers.js#the-ethers-project

File: docs/connectors-reference/voting-app.md [hunk 13]
```diff
@@ -12,11 +12,11 @@ import { Voting } from '@aragon/connect-thegraph-voting'
 const voting = new Voting(VOTING_APP_ADDRESS, VOTING_APP_SUBGRAPH_URL)
 ```
 
-Once you have a an instance of the `Voting` object, you can use the API below to fetch the data.
+Once you have a an instance of the `TokenManager` object, you can use the following API to fetch data.
```
Comment (0xGabi) [inline docs/connectors-reference/voting-app.md:15]:
```suggestion
Once you have an instance of the `Voting` object, you can use the following API to fetch data.

**Pull request merged**